static/.gitignore

```diff
@@ -1,2 +1,10 @@
 min/
 source-map/
+/js/bundle.js
+/generated/emoji
```

Comment:
Arguably, this section should just be `/generated`; I guess the reason we don't do that is `static/generated/README.md` is in Git.  Fine, I guess it needs to stay this way.